src/lib/apiErrorHandler.js

```diff
@@ -35,7 +36,8 @@ export const handleError = async (req, res, error, logContext) => {
   if (error instanceof Sequelize.Error) {
     await handleSequelizeError(req, res, error, logContext);
   } else {
-    logger.error(`${logContext.namespace} - UNEXPECTED ERROR - ${error.stack}`);
+    const err = error?.stack || error?.message || error;
```

Comment:
I don't know if you want to kill the if statement and just log the whole error here as well?

Reply:
ah yep, thanks for catching, missed this one (b9b320a)